### PR TITLE
Update properties.gd

### PR DESCRIPTION
**Pull request by scgm0**

### CODE REVIEW

File: addons/property_panel/properties.gd [hunk 1]
```diff
@@ -136,17 +136,17 @@ class RangeProperty extends Property:
 
 
 class IntProperty extends RangeProperty:
-	func _init(_name,_from,_to,_default = _from):
+	func _init(_name,_from=0,_to=100,_default = _from,_step = _from):
```
Comment (Jummit) [inline addons/property_panel/properties.gd]: I don't think `_from` is a good default for `_step` here, 1 would make more sense.

File: addons/property_panel/properties.gd [hunk 2]
```diff
@@ -136,17 +136,17 @@ class RangeProperty extends Property:
...
 		@warning_ignore("return_value_discarded")
-		super(_name, 1, _default)
+		super(_name, _default, _step)
 		from = _from
 		to = _to
 
 
 class FloatProperty extends RangeProperty:
-	func _init(_name,_from=0.0,_to=1.0,_default = _from):
+	func _init(_name,_from=0.0,_to=1.0,_default = _from,_step = _from):
```
Comment (Jummit) [inline addons/property_panel/properties.gd]: Here 0.1 would be a sensible default for `_step`.

**Review by Jummit — COMMENTED** [review]
Thanks for the improvements!

File: addons/property_panel/properties.gd (hunk 2 above)
Comment (scgm0) [reply addons/property_panel/properties.gd]: godot编辑器或许是0？以此将step默认为设置可更改的最小值

File: addons/property_panel/path_picker_button/path_picker_button.gd [hunk 3]
```diff
@@ -16,7 +16,7 @@ var path : String:
 		@warning_ignore("return_value_discarded")
 		emit_signal("changed")
 ## See [member FileDialog.filters].
```
Comment (Jummit) [inline addons/property_panel/path_picker_button/path_picker_button.gd]: This comment should be changed to something like "Members and their values set on new [FileDialog]s."

File: addons/property_panel/path_picker_button/path_picker_button.gd [hunk 4]
```diff
@@ -16,7 +16,7 @@ var path : String:
...
-var filters : PackedStringArray
+var option : Dictionary
```
Comment (Jummit) [inline addons/property_panel/path_picker_button/path_picker_button.gd]: I think `options` makes more sense.

**Review by Jummit — CHANGES_REQUESTED** [review]
Thanks for the changes! Just some small tweaks, then it's good to go.

File: addons/property_panel/path_picker_button/path_picker_button.gd (hunk 3 above)
Comment (scgm0) [reply addons/property_panel/path_picker_button/path_picker_button.gd]: My English is terrible, how about using "Members and their values set on new [FileDialog]s."?
(This is the reply after the machine flip)

Comment (Jummit) [reply addons/property_panel/path_picker_button/path_picker_button.gd]: Yes, sounds great! Whatever that machine is, it's working.

**Comment by Jummit** [comment]
Thanks so much for the changes!

**Pull request merged**